railties/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Use numeric UID and GID in Dockerfile template
+
+    The Dockerfile generated by `rails new` sets the default user and group
+    by name instead of UID:GID. This can cause the following error in kubernetes:
+    ```
+    container has runAsNonRoot and image has non-numeric user (rails), cannot verify user is non-root
+    ```
```

Comment:
There should be a line-break between the code block and other paragraphs, but it's not strictly enforced. In case anyone wants to submit a PR to fix that. :pray:

Reply:
My bad, fixed in 296664d59e4c65664de8928bd0d5bb68c1905573